README.md

```diff
@@ -3,15 +3,20 @@ KafkaFlow Retry is a .NET framework to retry messages on consumers, simple to us
 
 KafkaFlow Retry is an extention of [Kafka Flow](https://github.com/Farfetch/kafka-flow).
 
-## KafkaFlow Retry
 
-## Features
- - Simple Retry
- - Forever Retry
- - Durable Retry
- - Fluent configuration
- - Admin Web API to manage messages and queue messages
- - Persistence in SQL Server and MongoDb
+# Resilience policies
+
+|Policy| Description | Aka| Required Packages|
+| ------------- | ------------- |:-------------: |------------- |
+|**Simple Retry** <br/>(policy family)<br/><sub>([quickstart](#simple)&nbsp;;&nbsp;deep)</sub>|Many faults are transient and may self-correct after a short delay.| "Maybe it's just a blip" |   KafkaFlow.Retry |
+|**Forever Retry**<br/>(policy family)<br/><sub>([quickstart](#forever)&nbsp;;&nbsp;deep)</sub>|Many faults are semi-transient and may self-correct after multiple retries. | "Never give up" | KafkaFlow.Retry | 
+|**Durable Retry**<br/><sub>([quickstart](#durable)&nbsp;;&nbsp;deep)</sub>|Beyond a certain amount of retries and wait, you want to keep processing next-in-line messages but you can't loss the current offset message. As persistance databases, MongoDb or SqlServer are available. And you can manage in-retry messages through HTTP API.| "I can't stop processing messages but I can't loss messages"  | KafkaFlow.Retry <br/>KafkaFlow.Retry.API<br/><br/>KafkaFlow.Retry.SqlServer<br/>or<br/>KafkaFlow.Retry.MongoDb | 
+
+# Installing via NuGet
+Install packages related to your context. The Core package is required for all other packages. 
+
+## Requirements
+**.NET Core 2.1 and later using Hosted Service**
```

Comment:
should we change to .NET Core 3.1?

Reply:
As we spoke we will keep equivalent to KafkaFlow requirements.